locomotion-sdk/src/Components/PageHeader/styled.js

```diff
@@ -28,8 +29,21 @@ export const HeaderIcon = styled.Image`
 `;
 
 export const Header = styled.View`
-  padding: 15px;
   background-color: ${({ theme }) => theme.pageBackgroundColor};
   width: 100%;
   box-shadow: 0px 5px 3px rgba(68, 68, 68, 0.1);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
```

Comment:
did you test the other main pages for this change?

Reply:
yep